### PR TITLE
Add a coreml conversion script 

### PR DESCRIPTION
**Pull request by guillaumebrg**
This PR contains @marktodorovich's conversion script. Main difference with the original version is that I changed which parameters are exposed or hardcoded and I did some small refactoring changes early in the script, before the config file is parsed, so that it is possible to combine a generic backbone network with a specific classifier. After the config file is parsed, everything is pretty much the same.
 
Here is how to use it:
```
python scripts/conversion/convert_to_coreml.py --backbone=efficientnet --classifier=efficient_net_gesture_control --output_name=efficientnet_gesture_control --float16
```

For now, `--backbone=efficientnet` and `--classifier=efficient_net_gesture_control` are the only available options but it should be straightforward to later add support to more backbone networks (e.g. mobilenet) or other classifiers (e.g. fitness detector). I also think it won't be too hard to extend this to support custom classifiers once we have merged that finetuning script (#10)

### CODE REVIEW

File: scripts/conversion/convert_to_coreml.py [hunk 1]
```diff
@@ -0,0 +1,1175 @@
...
+}
+
+SUPPORTED_BACKBONE_CONVERSIONS = {
+    'efficientnet':
+        {
+            'config_file': 'scripts/conversion/cfg/efficientnet.cfg',
+            'weights_file': 'resources/strided_inflated_efficientnet.ckpt',
+            'model_name': 'strided_inflated_efficientnet',
+            'conversion_parameters': {**DEFAULT_CONVERSION_PARAMETERS, 'image_scale': 255.}
+        }
```
Comment (guillaumebrg) [inline scripts/conversion/convert_to_coreml.py:72]: This is where we can define more backbone networks

File: scripts/conversion/convert_to_coreml.py [hunk 2]
```diff
@@ -0,0 +1,1175 @@
...
+}
+
+SUPPORTED_CLASSIFIER_CONVERSIONS = {
+    'efficient_net_gesture_control':
+        {
+            'config_file': 'scripts/conversion/cfg/logistic_regression.cfg',
+            'placeholder_values': {'NUM_CLASSES': '30'},
+            'weights_file': 'resources/gesture_detection/efficientnet_logistic_regression.ckpt',
+            'corresponding_backbone': 'efficientnet',
+        }
```
Comment (guillaumebrg) [inline scripts/conversion/convert_to_coreml.py:82]: This where we can add more classifiers

File: scripts/conversion/convert_to_coreml.py [hunk 3]
```diff
@@ -0,0 +1,1175 @@
...
+                output_stream.write(line)
+
+    output_stream.seek(0)
+    return output_stream
+
+
+def convert(backbone_settings, classifier_settings, output_name, float16, plot_model):
+    output_dir = 'resources/coreml/'
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
```
Comment (antoinemrcr) [inline scripts/conversion/convert_to_coreml.py:119]: os.makedirs(output_dir, exist_ok=True)  
instead of the two lines ?

**Comment by antoinemrcr** [comment]
Can you add the "Here is how to use it:" in the readme ?

File: README.md [hunk 4]
```diff
@@ -119,6 +119,16 @@ The estimated calorie estimates are roughly in the range produced by wearable de
 From our experiments, our estimates correlate well with the workout intensity (intense workouts burn more calories) so, regardless of the absolute accuracy, it should be fair to use this metric to compare one workout to another.
 
 
+## Running on an iOS Device and CoreML Conversion
+
+If you're interested in mobile app development and want to run our models on iOS devices, please check out [20bn-realtimenet-ios](https://github.com/TwentyBN/20bn-realtimenet-iOS).
+This other repo provides step by step instructions on how to get our gesture demo to run on an iOS device.
```
Comment (sunny-panchal) [inline README.md:125]:
```suggestion
If you're interested in mobile app development and want to run our models on iOS devices, please check out [20bn-realtimenet-ios](https://github.com/TwentyBN/20bn-realtimenet-iOS) for step by step instructions on how to get our gesture demo to run on an iOS device.
```

**Pull request merged**